ss58-registry.json

```diff
@@ -498,6 +498,15 @@
     },
     {
       "prefix": 67,
+      "network": "genshiro",
+      "displayName": "Genshiro Network",
+      "symbols": ["GENS", "EQD", "LPT0"],
+      "decimals": [9, 9, 9],
+      "standardAccount": "*25519",
+      "website": "https://genshiro.equilibrium.io"
+    },
+    {
+      "prefix": 68,
       "network": "equilibrium",
       "displayName": "Equilibrium Network",
       "symbols": ["Unknown", "USD", "EQ", "ETH", "BTC", "EOS", "DOT", "CRV"],
```

Comment:
@joepetrowski Following your comments, wouldn't it be also recommended to list only chain native tokens for the Equilibrium network then?